test/unit/lib/plugins/aws/package/compile/events/schedule.test.js

```diff
@@ -45,6 +49,11 @@ describe('AwsCompileScheduledEvents', () => {
             {
               schedule: 'cron(5,35 12 ? * 6l 2002-2005)',
             },
+            {
```

Comment:
This specific test should be already written with `runServerless` approach. While adjusting the old tests like above with making the parameter into an array is okay, we definitely don't want to introduce new cases using the old approach. Do you think you could cover this case with `runServerless`-based approach?

Reply:
Sure, I'll update this test case then :+1: .

Reply:
Done. I've updated all tests in this file to use `runServerless`.